<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,3,1">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QToolButton" name="btnAbout">
          <property name="toolTip">
           <string>About</string>
          </property>
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="btnMinus">
          <property name="toolTip">
           <string>-1 Hz</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnPlus">
          <property name="toolTip">
           <string>+1 Hz</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDown">
          <property name="toolTip">
           <string>-25 Hz</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnUp">
          <property name="toolTip">
           <string>+25 Hz</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="LCDNumber" name="lcdNumber">
          <property name="smallDecimalPoint">
           <bool>false</bool>
          </property>
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QToolButton" name="btnPower">
          <property name="toolTip">
           <string>On</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbRadio">
          <property name="toolTip">
           <string>Device</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkAgc">
          <property name="toolTip">
           <string>Automatic gain control On/Off</string>
          </property>
          <property name="text">
           <string>AGC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkStereo">
          <property name="toolTip">
           <string>Stereo On/Off</string>
          </property>
          <property name="text">
           <string>Stereo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LEDButton" name="led">
          <property name="toolTip">
           <string>Stereo</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LCDNumber</class>
   <extends>QLCDNumber</extends>
   <header>gui/MainWindow.h</header>
  </customwidget>
  <customwidget>
   <class>LEDButton</class>
   <extends>QToolButton</extends>
   <header>gui/LEDButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
